src/helpers/issue.ts

```diff
@@ -95,6 +95,32 @@ export const listAllIssuesForRepo = async (state: "open" | "closed" | "all" = "o
   return issuesArr;
 };
 
+export const getIncentivizedUsers = async (issue_number: number) => {
+  const comments = await getAllIssueComments(issue_number);
+  const incentiveComments = comments.filter((comment) => comment.body.startsWith("/comment-incentives"));
+  let users: { [key: string]: boolean } = {};
+  for (const incentiveComment of incentiveComments) {
+    const parts = incentiveComment.body.split(" ");
+    parts.shift();
+    if (parts.pop() == "true") {
```

Comment:
this relies on command order. just use a for loop and check for `true`, `false`, or starts with `@` (there shouldn't more than one `true` or `false`). you can look at this [example](https://github.com/ubiquity-whilefoo/ubiquibot/blob/3056bc91e062ab9e1b161d34d2da1418a3771a30/src/handlers/comment/handlers/allow.ts#L27)